app/services/audit/catalog_to_moab.rb

```diff
@@ -69,26 +74,25 @@ def online_moab_found?
     #   report results (and return them)
     def compare_version_and_take_action
       moab_version = moab.current_version_id
-      results.actual_version = moab_version
```

Comment:
This was a smell: Having to set the version using an accessor instead of at initialization.